### ISSUE TITLE
DL3059 does not trigger if consecutive `RUN` instructions are separated by a comment

### ISSUE THREAD

**Issue by Kaligule** [hadolint/hadolint#712]
- [x] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

(I guess this is somewhere in the middle of bug report and feature request. You decide).

### Expected behavior

Hadolint should find two consecutive `RUN` commands (even if there are comments in between them) and cite [DL3059](https://github.com/hadolint/hadolint/wiki/DL3059):

    /Dockerfile:5 DL3059 info: Multiple consecutive `RUN` instructions. Consider consolidation.

### Actual behavior

Because of the comment separating the `RUN` instructions the DL3059 is not triggered. I don't get any output and hadolint exits successfully (with statuscode `0`)

### Steps to reproduce the behavior

Run hadolint on the  following minimal-example Dockerfile below

```Dockerfile
FROM alpine:3.4

RUN apk update
# install my favorite editor
RUN apk add emacs
```

**Output of `hadolint --version`**

```bash
Haskell Dockerfile Linter v2.7.0-0-g3e13ec6
```

**Notes**

The code should be adopted here: https://github.com/hadolint/hadolint/blob/dcf946a769a831c86e7c7e234e67cbfc6abc9442/src/Hadolint/Rule/DL3059.hs#L19-L22
The problem occurs because `language-docker` does recognize `Comment` as an instruction: https://github.com/hadolint/language-docker/blob/a1e8ebf778f7b7999711e6c09ff537a0c2594345/src/Language/Docker/Syntax.hs#L357

**Comment by m-ildefons** [comment]
Hi @Kaligule ,
thanks for the bug report. This case indeed was an oversight on my part, when I wrote that rule.
